tests/lcs/core/engine/debugger.livecodescript

```diff
@@ -35,3 +35,43 @@ on TestWatchedVariables
    set the watchedVariables to tWatches
    TestAssert "watchedVariables handler local round trips", the watchedVariables is tWatches
 end TestWatchedVariables
+
+
+command _TestErrorInSwitchCaseExpression
+    switch 1
+        case foobar()
+        break
+    end switch
+end _TestErrorInSwitchCaseExpression
+
+command _TestErrorInSwitchCaseExpressionContdLine
+    switch 1
+        case \
+            foobar()
+        break
+    end switch
+end _TestErrorInSwitchCaseExpressionContdLine
+
+command _TestErrorInIfCaseExpressionContdLine
+    if \
+        foobar() then
+    end if
+end _TestErrorInIfCaseExpressionContdLine
+
+command _TestErrorInForCaseExpressionContdLine
+    repeat \
+        foobar()
+    end repeat
+end _TestErrorInForCaseExpressionContdLine
+
+command _TestErrorInForWithExpressionContdLine
+    repeat with \
```

Comment:
Currently this one will not compile - `repeat with <expression>` is invalid. You probably want 
```
repeat with x = 1 to 10 \
   step foobar()
```
and 
```
repeat with x = 1 to \
   foobar()
```

and 

```
repeat until \
   foobar()
```
etc.

Do we need add a line number param to the test assert throw api?